feature-ui-solve-problem/src/main/kotlin/team/duckie/app/android/feature/ui/solve/problem/common/TopBar.kt

```diff
@@ -38,6 +49,40 @@ internal fun CloseAndPageTopBar(
     )
 }
 
+@Composable
+internal fun TimerTopBar(
+    modifier: Modifier = Modifier,
+    onCloseClick: () -> Unit,
+    progress: () -> Float,
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(horizontal = 16.dp),
+    ) {
+        QuackTopAppBar(
+            modifier = modifier,
```

Comment:
```suggestion
            modifier = Modifier,
```
modifier가 이미 상위 컴포저블에 들어가 있네요!